hooks/pre-command

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -euo pipefail
+
+# This hook only runs if the hook parameter is set to "pre-command".
+
+if [[ "${BUILDKITE_PLUGIN_AWS_ASSUME_ROLE_WITH_WEB_IDENTITY_HOOK:-}" == "pre-command" ]]; then
+	echo "Running in the pre-command hook"
+else
+	echo "Skipping pre-command hook"
+	exit 0
+fi
+
+DIR="$(cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd)"
+
+"$DIR/../lib/plugin.bash"
```

Comment:
based on the latest test failures, everything indicates you were right to source this in the first place and I led you astray... my bad 😞